src/main/java/org/mockito/internal/progress/MockingProgressImpl.java

```diff
@@ -142,15 +145,20 @@ public ArgumentMatcherStorage getArgumentMatcherStorage() {
     }
 
     public void mockingStarted(Object mock, MockCreationSettings settings) {
-        //TODO 384 listeners
-        if (listener instanceof MockCreationListener) {
-            ((MockCreationListener) listener).mockCreated(mock, settings);
+        for (MockitoListener listener : listeners) {
+            if (listener instanceof MockCreationListener) {
+                ((MockCreationListener) listener).mockCreated(mock, settings);
+            }
         }
         validateMostStuff();
     }
 
-    public void setListener(MockitoListener listener) {
-        this.listener = listener;
+    public void addListener(MockitoListener listener) {
```

Comment:
null arguments should not be allowed, they may cause an NPE later when listeners are notified